synthesizer/process/src/finalize.rs

```diff
@@ -104,7 +104,11 @@ impl<N: Network> Process<N> {
         lap!(timer, "Verify the number of transitions");
 
         // Construct the call graph.
-        let call_graph = self.construct_call_graph(execution)?;
+        let call_graph = if state.block_height() >= N::CONSENSUS_V3_HEIGHT {
+            Default::default()
```

Comment:
Would `None` be clearer?

Reply:
The value here should be an empty hash map. 
Are you suggesting to define `call_graph` on L188 as an `Option<HashMap<N::TransitionID, Vec<N::TransitionID>>>`?

Reply:
I'm not sure, I feel like adding the `Option` requires changing the interface and destructuring later.
No strong opinions so I'll defer to reviewers recommendation.

Reply:
1. I think `Default::default()` is fine here to reduce LoC changed.
2. I would prefer a match case instead of a `let else` structure. Just to be consistent with how we did it with `CONSENSUS_V2_HEIGHT`. Example below:


```
    match block_height < N::CONSENSUS_V2_HEIGHT {
        true => coinbase_reward_v1(...),
        false => coinbase_reward_v2(...),
    }
```